components/automate-ui/src/app/pages/team-management/team-management.component.spec.ts

```diff
@@ -196,7 +197,7 @@ describe('TeamManagementComponent', () => {
       component.openCreateModal();
       component.createTeamForm.controls['name'].setValue(team.name);
       component.createTeamForm.controls['id'].setValue(team.id);
-      component.createV2Team();
+      component.createV2Team([]);
```

Comment:
why does this need an empty array?

Reply:
It's just the array of projects. Nothing else in here is testing project specific stuff so I left it empty.